web/locales/de.json

```
@@ -104,7 +104,8 @@
         "database-ad": "Auf der Suche nach historischen Daten? <a href=\"https://data.electricitymap.org?utm_source=electricitymap.org&utm_medium=referral\" target=\"_blank\">Sieh dir unsere Datenbank an!</a>",
         "api-ad": "Benötigst du Live-Daten für deine Apps oder Geräte? <a href=\"https://api.electricitymap.org?utm_source=electricitymap.org&utm_medium=referral\" target=\"_blank\">Probiere unsere API!</a>",
         "legend": "Legende",
-        "faq": "Häufig gestellte Fragen"
+        "faq": "Häufig gestellte Fragen",
+        "webgl-not-supported": "Diese Karte kann nicht angezeigt, weil dieser Webbrowser kein WebGL unterstützt."
```

I think there is a typo here :slightly_smiling_face: 

```suggestion
        "webgl-not-supported": "Diese Karte kann nicht angezeigt werden, weil dieser Webbrowser kein WebGL unterstützt."
```